### DISCUSSION TITLE
Janet to Javascript

### DISCUSSION THREAD

**Discussion by bentxt** [janet-lang/janet#686] [Ideas]
Hi fellow Janet users

Let me first thank you for Janet. For my daily scripting needs I find myself more and more using Janet and I like it very much. Previously I was using a mix of Shell scripts, Perl and (Gauche) Scheme, but I had to admit that Janet shows itself more useful for mundane daily scripting.

Now the other language I seek to abandon is Javascript. There are already two libraries 

- https://github.com/ahungry/ahungry-janet-to-js
- https://github.com/staab/janet-js

They seem a bit dormant, so I'd like to have your general opinion in having a Janet -> JS compiler. Do you think that's sensible thing to do? What do you think to have it and support in the core compiler?

**Comment by wrq** [comment]
It's already possible to get Janet running on WASM. Personally, I think the idea is pretty neat, but I doubt there's enough resources to warrant core developers working on it in the main branch. 

On top of that, it seems rather needless to compete in this space when there's already Clojure/Script. That might be a hotter take though.
